docs/GETTING_STARTED.rst

```diff
@@ -0,0 +1,274 @@
+Getting Started
+=================
+
+.. figure:: ../res/Scality-S3-Server-Logo-Large.png
+   :alt: S3 Server logo
+
+|CircleCI| |Scality CI|
+
+Contributing
+------------
+
+In order to contribute, please follow the `Contributing
+Guidelines <https://github.com/scality/Guidelines/blob/master/CONTRIBUTING.md>`__.
+
+Installation
+------------
+
+Dependencies
```

Comment:
I would add a title here, same level as all the "run it with X backend", called "Requirements", and that would have "dependencies", "clone source code" and "install js dependencies" as sub-titles.